### PR TITLE
[backend/frontend] Fix inject test condition

### PR DESCRIPTION
**Pull request by RomuDeuxfois**
- Reduce time execution for bulk test inject
- Remove check on teams -> when we test an inject, emails or SMS are send to the current user. Teams are not used.
- Remove InjectTestStatus on Inject class -> is not used

### CODE REVIEW

**Review by johanah29 — COMMENTED** [review]
image
This error message prompts when the user tries to test a technical inject using the bulk actions toolbar. Can we have a more appropriate message ?

**Comment by RomuDeuxfois** [comment]
> image This error message prompts when the user tries to test a technical inject using the bulk actions toolbar. Can we have a more appropriate message ?

Fix with "This inject can't be test"

File: openbas-front/src/actions/injects/inject-action.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const testInject = (injectId: string) => {
 export const bulkTestInjects = (injectIds: string[]) => {
   const data = injectIds;
   const uri = '/api/injects/bulk/test';
-  return simplePostCall(uri, data, 'The inject does not have any team defined');
+  return simplePostCall(uri, data, "This inject can't be test");
```
Comment (damgouj) [inline openbas-front/src/actions/injects/inject-action.ts:14]: can't be tested ?

**Review by damgouj — APPROVED** [review]

File: openbas-front/src/actions/injects/inject-action.ts (hunk 1 above)
Comment (guillaumejparis) [reply openbas-front/src/actions/injects/inject-action.ts:14]: missing translation

**Comment by guillaumejparis** [comment]
This message does not sense anymore ,no ?
`Only SMS and emails related injects with teams defined will be tested`

**Pull request merged**